charts/nri-kube-events/ci/test-values.yaml

```diff
@@ -8,6 +8,39 @@ sinks:
   # The newRelicInfra sink sends all events to New relic.
   newRelicInfra: true
 
+config:
+  accountID: 111
+  region: EU
+
+rbac:
+  create: true
+
+runAsUser: 1000
+
```

Comment:
NIT: Do this apply to this chart?

Reply:
Nope, I don't know why I really add it... Updated in 1075616da92089379b613d64df96f24fbd82066b.